src/App.tsx

```diff
@@ -8,55 +8,148 @@ import { PostsList } from './components/PostsList';
 import { PostDetails } from './components/PostDetails';
 import { UserSelector } from './components/UserSelector';
 import { Loader } from './components/Loader';
+import { getUsers } from './api/users';
+import { User } from './types/User';
+import { getPosts } from './api/posts';
+import { Post } from './types/Post';
+import { getComments } from './api/commets';
+import { Comment } from './types/Comment';
 
 export const App: React.FC = () => {
+  const [userSelect, setUserSelect] = useState<User[]>([]);
+  const [selectedUser, setSelectedUser] = useState<User | null>(null);
+
+  const [isPostError, setIsPostError] = useState<boolean>(false);
+  const [isPostsLoading, setIsPostsLoading] = useState<boolean>(false);
```

Comment:
You don't need to specify that it's boolean in such cases. TypeScript understands it itself.